templates/web/base/report/new/category_groups.html

```diff
@@ -0,0 +1,14 @@
+<select class="form-control js-grouped-select" name='category' id='form_category' data-role='[% c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]' data-body='[% c.user.from_body.name %]'>
```

Comment:
Splitting them up into separate files makes it even harder to keep them in sync :)